contrib/dev/Vagrantfile

```diff
@@ -99,19 +111,25 @@ def populate_os_config(os, ip)
       address: '192.168.10.10',
     },
   ]
+  os.security_groups = [ 'skydive' ]
+end
+
+def box_url(type)
+  "https://download.fedoraproject.org/pub/fedora/linux/releases/27/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-27-1.6.x86_64.vagrant-#{type}.box"
 end
 
 Vagrant.configure(2) do |config|
   config.vm.define "dev" do |dev|
     dev.vm.hostname = "dev"
     dev.vm.synced_folder ".", "/vagrant", disabled: true
-    dev.vm.synced_folder SYNCED_FOLDER, "/home/vagrant/src/github.com/skydive-project/skydive", disabled: false, :type => "nfs", :nfs_udp => false
+    dev.vm.synced_folder SKYDIVE_HOST, SKYDIVE_GUEST, disabled: false, :type => "nfs", :nfs_udp => false
+    dev.vm.synced_folder ANSIBLE_HOST, ANSIBLE_GUEST, disabled: false, :type => "nfs", :nfs_udp => false
```

Comment:
Why exporting 2 folders as the ansible playbooks are in the Skydive repository ?

Reply:
Because in my user case I am using a development tree which is not identical to were we run `devmode` from, so I always want to take the ansible scripts from the location were I ran 'vagrant up` while the `skydive` folder can either default to `../..` or `$GOPATH/src/github.com/skydive-project/skydive` or (in the works) nothing ... as `vboxfs` is very slow suggesting that sometimes just working with a local `git clone github.com/skydive-project/skydive` is best.

Reply:
Ok, I'm fine with the 2 folders